### DISCUSSION TITLE
2FA restoration

### DISCUSSION THREAD

**Discussion by bprbots** [community/community#125943] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, My mobile phone was stolen some time ago and I had activated 2FA on my account. Now that I bought a new phone, I can neither receive the code through Google Authenticator, not can I access my account through GitHub.
I lost the recovery codes, I don't have SSH key and personal access token. But I have access to email, password, phone number and any other information. Please help me to access my account
In the GitHub support section, when I click on the "New Ticket" button, it sends me to the FAQ section.. Please help. 💔

**Comment by davevad93** [comment]
Hi @bprbots ,

Try with raising a ticket [here](https://support.github.com/contact/account) and select one between `Two-factor authentication recovery` or `Sign in, verification or other account questions`.

**Reply by bprbots** [reply]
thanks, I will try your solution 🤍

**Reply by davevad93** [reply]
You're welcome @bprbots , I hope you can solve your issue!
